internal/controllers/clustertemplate_controller.go

```diff
@@ -327,6 +334,54 @@ func generateTemplateID(ctx context.Context, c client.Client, object *provisioni
 	return nil
 }
 
+// validateTemplateParameterSchema return true if the schema contained in the templateParameterSchema
+// field contains the required mandatory parameters
+// - nodeClusterName
+// - oCloudSiteId
+// - policyTemplateParameters
+// - clusterInstanceParameters
+func validateTemplateParameterSchema(object *provisioningv1alpha1.ClusterTemplate) error {
+	mandatoryParams := []string{"nodeClusterName",
```

Comment:
All these values will probably be standardized, so they can be made constants.

Reply:
There are existing constants for these values.

Reply:
Fixed

Reply:
Fixed